.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
```

Comment:
While I was writing it I realized that an entire section wouldn't be a bad idea, specifically because it mentions that you need to add new tests (which is something that can be easily forgotten unfortunately). Also it would give you the area where to add BackstopJS screenshots if needed.

Those are the pros, the cons as already said is that it could be too much